clang/lib/Driver/Types.cpp

```diff
@@ -175,14 +175,15 @@ bool types::isAcceptedByFlang(ID Id) {
   case TY_PP_F_FreeForm:
   case TY_F_FixedForm:
   case TY_PP_F_FixedForm:
+    return true;
 #else
   case TY_Fortran:
   case TY_PP_Fortran:
-#endif
     return true;
```

Comment:
Should the code in line 182 be removed?

Reply:
No, it shouldn't. Doing that would create an unnecessary diff relative to upstream `release/15.x`.